src/test/java/WaiterTest.java

```diff
@@ -0,0 +1,78 @@
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+
+public class WaiterTest {
+    WebDriver driver;
+
+    @BeforeEach
+    public void openBrowser() {
+        driver = new ChromeDriver();
+    }
+
+    @ParameterizedTest
+    @MethodSource("person")
+    void Names(String login, String password) throws InterruptedException {
```

Comment:
Give normal name to the test and naming must start with lower case

Reply:
done